utils/text/sanitizers.py

```diff
@@ -0,0 +1,32 @@
+# Python Standard Library Imports
+import html
+import re
+
+
+# isort: off
+
+
+def sanitize_cookie_value(value: str) -> str:
+    """Sanitize Cookie Value
+
+    Sanitizes a cookie value by escaping HTML special characters and
+    removing non-alphanumeric characters, except for some safe ones like
+    hyphens and underscores.
+
+    Args:
+    - value (str): The cookie value to be sanitized.
+
+    Returns:
+    - str: The sanitized cookie value.
+
+    References:
+    - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie
+    - https://stackoverflow.com/a/1969339
+    """
+    # Escape HTML special characters to prevent XSS
+    sanitized_value = html.escape(value)
```

Comment:
Really, in a cookie? Honest question.

Reply:
It's suggested by ChatGPT and It doesn't hurt to do extra checks to ensure no funny business going on.